ui/common-lib/package.yaml

```diff
@@ -0,0 +1,32 @@
+<<: !include "../../hpack/module.yaml"
+
+name: ariadne-common-ui-lib
+
+library:
+  <<: *lib-common
+  ghc-options:
+    - -Wno-unused-top-binds
+  dependencies:
+    - Earley
+    - Hclip
+    - ariadne-core
+    - ariadne-cardano
```

Comment:
This dependency looks wrong.

Reply:
I see that it's needed for common glue. However, `ariadne-vty-lib` must not depend on any Cardano stuff, because it's also used in Disciplina.
The most obvious way to avoid it is to have 2 common libs (one with common face and the other one with common glue). I find it a bit disappointing to have 2 libs with one module, but can't quickly come up with a better approach.

Reply:
I see the problem. Is this also why app and lib are different packages?